tensorboard/plugins/histogram/histograms_demo.py

```diff
@@ -29,40 +31,42 @@ def run_all(logdir, verbose=False):
   """Generate a bunch of histogram data, and write it to logdir."""
   del verbose
 
+  tf.set_random_seed(0)
+
   k = tf.placeholder(tf.float32)
 
   # Make a normal distribution, with a shifting mean
   mean_moving_normal = tf.random_normal(shape=[1000], mean=(5*k), stddev=1)
   # Record that distribution into a histogram summary
-  tf.summary.histogram("normal/moving_mean", mean_moving_normal)
+  histogram_summary.op("normal/moving_mean", mean_moving_normal)
```

Comment:
Agreed; I think that eventually we're going to end up using
```py
from tensorboard.plugins import histogram
from tensorboard.plugins import scalar
from tensorboard.plugins import text

histogram.op(...)
scalar.pb(...)
```
which will require writing an `__init__` file for the `tensorboard.plugins.histogram` package that re-exports `op` and `pb` from this file.

(One problem with exposing a `tensorboard.summary` package is that it inverts the hierarchy: plugins define their summaries, not the other way around, and if an external contributor develops a plugin `foo` then it will be `bigcorp.foo_plugin.summary` and not `bigcorp.summary.foo_plugin`. We want plugins' code and APIs to be self-contained in one directory per plugin.)